### PR TITLE
Forms and helpers

### PR DESCRIPTION
**Pull request by WCanirinka**
The project is about creating forms and helpers by creating a rails test application.

### CODE REVIEW

**Comment by WCanirinka** [comment]
After applying linters to the project, we are now requesting for a code review for the project

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+         <ul>
+         <% @user.errors.full_messages.each do |msg| %>
+           <li><%= msg %></li>
+         <% end %>
+         </ul>
+       </div>
+     <% end %>
+
+    <%= form_for(@user) do |f| %>
```
Comment (Ispirett) [inline app/views/users/new.html.erb:44]: !: You are missing the plan HTML form [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) 
2: form tag form is missing [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png)

**Review by Ispirett — CHANGES_REQUESTED** [review]
## status: Changes required :1st_place_medal: 
Great Job so far @WCanirinka  @300ms you done some great work.

There are just are few forms missing please see below.

**Comment by WCanirinka** [comment]
We have applied the necessary changes and we are requesting for a new code review.

**Review by WCanirinka — COMMENTED** [review]
Requesting for a review

**Review by mosaaleb — APPROVED** [review]
### Approved :clap:

#### You can merge :+1: 

##### Muhammad Ebeid

**Pull request merged**